RecoTracker/IterativeTracking/python/TobTecStep_cff.py

```diff
@@ -233,16 +255,25 @@
     TrajectoryBuilderPSet = cms.PSet(refToPSet_ = cms.string('tobTecStepTrajectoryBuilder')),
     doSeedingRegionRebuilding = True,
     useHitsSplitting = True,
-    cleanTrajectoryAfterInOut = True
+    cleanTrajectoryAfterInOut = True,
+    TrajectoryCleaner = 'tobTecStepTrajectoryCleanerBySharedHits'
```

Comment:
Also here, TrajectoryCleaner is moved inside the definition of tobTecStepTrackCandidates
Here it doesn't seems to touch Phase2 or anything else